Source/UnitTests/Core/IOS/ES/FormatsTest.cpp

```diff
@@ -0,0 +1,227 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include <gtest/gtest.h>
+
+#include <vector>
+
+#include "Core/IOS/ES/Formats.h"
+#include "TestBinaryData.h"
```

Comment:
Tried this; it doesn't work and I'm not sure how to set up the include path to make it work. Right now, this is the only case where we need to include a header file; do you think it'd be worth trying to get that to work?